bench/rttanalysis: TestBenchmarkExpectation failed

bench/rttanalysis.TestBenchmarkExpectation [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3831142&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3831142&tab=artifacts#/) on master @ [2b8b0f64c63a376784926667f5fbffa4c4fcc60b](https://github.com/cockroachdb/cockroach/commits/2b8b0f64c63a376784926667f5fbffa4c4fcc60b):


```
           134.037ms      0.012ms                event:server/node.go:923 [n=1] node received request: 1 QueryIntent
           134.066ms      0.029ms                event:kv/kvserver/store_send.go:157 [n=1,s=1] executing QueryIntent [/Table/33/1/"rolea"/"NOLOGIN"/0,/Min), [txn: 7416219c]
           134.086ms      0.020ms                event:kv/kvserver/replica_send.go:153 [n=1,s=1,r=29/1:/Table/3{3-4}] read-only path
           134.114ms      0.028ms                event:kv/kvserver/concurrency/concurrency_manager.go:187 [n=1,s=1,r=29/1:/Table/3{3-4}] sequencing request
           134.130ms      0.016ms                event:kv/kvserver/concurrency/concurrency_manager.go:259 [n=1,s=1,r=29/1:/Table/3{3-4}] acquiring latches
           134.154ms      0.024ms                event:kv/kvserver/replica_read.go:242 [n=1,s=1,r=29/1:/Table/3{3-4}] executing read-only batch
           134.197ms      0.044ms                event:scan stats: stepped 0 times (0 internal); seeked 1 times (1 internal)
           134.240ms      0.043ms                event:scan stats: stepped 2 times (2 internal); seeked 2 times (2 internal)
           134.335ms      0.095ms                event:kv/kvserver/replica_evaluate.go:508 [n=1,s=1,r=29/1:/Table/3{3-4}] evaluated QueryIntent command header:<key:"\251\211\022rolea\000\001\022NOLOGIN\000\001\210" > txn:<id:7416219c-8713-4bda-9b74-f178ba598c0d key:"\210" write_timestamp:<wall_time:1638425372086132737 > min_timestamp:<wall_time:1638425372086132737 > priority:111830 sequence:3 > error_i..., txn="sql txn" meta={id=7416219c key=/Table/SystemConfigSpan/Start pri=0.00520749 epo=0 ts=1638425372.086132737,0 min=1638425372.086132737,0 seq=0} lock=true stat=PENDING rts=1638425372.086132737,0 wto=false gul=1638425372.586132737,0 : resp=header:<> found_intent:true, err=<nil>
           134.354ms      0.019ms                event:kv/kvserver/replica_read.go:178 [n=1,s=1,r=29/1:/Table/3{3-4}] read completed
           134.176ms     -0.178ms                structured:{"@type":"type.googleapis.com/cockroach.roachpb.ScanStats","numInterfaceSeeks":"1","numInternalSeeks":"1","numInterfaceSteps":"0","numInternalSteps":"0"}
           134.227ms      0.051ms                structured:{"@type":"type.googleapis.com/cockroach.roachpb.ScanStats","numInterfaceSeeks":"2","numInternalSeeks":"2","numInterfaceSteps":"2","numInternalSteps":"2"}
           142.063ms      8.045ms            event:kv/kvclient/kvcoord/range_iter.go:175 [n=1,client=127.0.0.1:37856,hostssl,user=root,txn=7416219c] querying next range at /Table/SystemConfigSpan/Start
           142.105ms      0.043ms            event:kv/kvclient/kvcoord/dist_sender.go:1945 [n=1,client=127.0.0.1:37856,hostssl,user=root,txn=7416219c] r6: sending batch 1 EndTxn to (n1,s1):1
           142.125ms      0.020ms            event:rpc/nodedialer/nodedialer.go:160 [n=1,client=127.0.0.1:37856,hostssl,user=root,txn=7416219c] sending request to local client
           142.135ms      0.009ms                === operation:/cockroach.roachpb.Internal/Batch _verbose:1 node:1 span.kind:server
           142.148ms      0.013ms                event:server/node.go:923 [n=1] node received request: 1 EndTxn
           142.186ms      0.038ms                event:kv/kvserver/store_send.go:157 [n=1,s=1] executing EndTxn(commit modified-span (system-config)) [/Table/SystemConfigSpan/Start], [txn: 7416219c]
           142.204ms      0.018ms                event:kv/kvserver/replica_send.go:157 [n=1,s=1,r=6/1:/Table/{SystemCon…-11}] read-write path
           142.233ms      0.029ms                event:kv/kvserver/concurrency/concurrency_manager.go:187 [n=1,s=1,r=6/1:/Table/{SystemCon…-11}] sequencing request
           142.246ms      0.013ms                event:kv/kvserver/concurrency/concurrency_manager.go:259 [n=1,s=1,r=6/1:/Table/{SystemCon…-11}] acquiring latches
           142.278ms      0.032ms                event:kv/kvserver/replica_write.go:144 [n=1,s=1,r=6/1:/Table/{SystemCon…-11}] applied timestamp cache
           142.295ms      0.017ms                event:kv/kvserver/replica_write.go:436 [n=1,s=1,r=6/1:/Table/{SystemCon…-11}] executing read-write batch
           142.336ms      0.041ms                event:scan stats: stepped 0 times (0 internal); seeked 1 times (1 internal)
           142.363ms      0.027ms                event:scan stats: stepped 0 times (0 internal); seeked 1 times (1 internal)
           142.902ms      0.539ms                event:kv/kvserver/replica_evaluate.go:508 [n=1,s=1,r=6/1:/Table/{SystemCon…-11}] evaluated EndTxn command header:<key:"\210" sequence:25 > commit:true deadline:<wall_time:1638425680360028253 > internal_commit_trigger:<modified_span_trigger:<system_config_span:true > > lock_spans:<key:"\213\211\214\212\211" > lock_spans:<key:"\213\211\251\212\211" > lock_spa..., txn="sql txn" meta={id=7416219c key=/Table/SystemConfigSpan/Start pri=0.00520749 epo=0 ts=1638425372.086132737,0 min=1638425372.086132737,0 seq=25} lock=true stat=PENDING rts=1638425372.086132737,0 wto=false gul=1638425372.586132737,0 : resp=header:<txn:<meta:<id:7416219c-8713-4bda-9b74-f178ba598c0d key:"\210" write_timestamp:<wall_time:1638425372086132737 > min_timestamp:<wall_time:1638425372086132737 > priority:111830 sequence:25 > name:"sql txn" status:COMMITTED last_heartbeat:<wall_time..., err=<nil>
           142.926ms      0.024ms                event:kv/kvserver/replica_proposal.go:833 [n=1,s=1,r=6/1:/Table/{SystemCon…-11}] need consensus on write batch with op count=5
           142.950ms      0.024ms                event:kv/kvserver/replica_raft.go:91 [n=1,s=1,r=6/1:/Table/{SystemCon…-11}] evaluated request
           142.968ms      0.018ms                event:kv/kvserver/replica_raft.go:135 [n=1,s=1,r=6/1:/Table/{SystemCon…-11}] proposing command to write 0 new keys, 0 new values, -4 new intents, write batch size=930 bytes
           142.322ms     -0.646ms                structured:{"@type":"type.googleapis.com/cockroach.roachpb.ScanStats","numInterfaceSeeks":"1","numInternalSeeks":"1","numInterfaceSteps":"0","numInternalSteps":"0"}
           142.347ms      0.025ms                structured:{"@type":"type.googleapis.com/cockroach.roachpb.ScanStats","numInterfaceSeeks":"1","numInternalSeeks":"1","numInterfaceSteps":"0","numInternalSteps":"0"}
           143.024ms      0.677ms                    === operation:raft application _verbose:1 node:1 range:6/1:/Table/{SystemCon…-11} store:1
           143.099ms      0.075ms                    event:kv/kvserver/replica_application_state_machine.go:467 [n=1,s=1,r=6/1:/Table/{SystemCon…-11}] applying command
           143.149ms      0.050ms                    event:scan stats: stepped 0 times (2 internal); seeked 2 times (2 internal)
           143.169ms      0.020ms                    event:scan stats: stepped 0 times (2 internal); seeked 2 times (2 internal)
           143.267ms      0.098ms                    event:kv/kvserver/replica_application_state_machine.go:1093 [n=1,s=1,r=6/1:/Table/{SystemCon…-11}] LocalResult (reply: (err: <nil>), *roachpb.EndTxnResponse, #encountered intents: 0, #acquired locks: 0, #resolved locks: 4#updated txns: 1 #end txns: 0, GossipFirstRange:false MaybeGossipSystemConfig:true MaybeGossipSystemConfigIfHaveFailure:false MaybeAddToSplitQueue:false MaybeGossipNodeLiveness:<nil>
           143.408ms      0.140ms                    event:scan stats: stepped 126 times (88 internal); seeked 2 times (2 internal)
           234.279ms     90.871ms                    event:kv/kvserver/replica_evaluate.go:508 [n=1,s=1,r=6/1:/Table/{SystemCon…-11}] evaluated Scan command header:<key:"\210" end_key:"\223" >, txn=<nil> : resp=header:<num_keys:62 num_bytes:29079 > rows:<key:"\213\211\211\212\211" value:<raw_bytes:"\363\276\033\351\003\0221\n\006system\020\001\032\035\n\t\n\005admin\0200\n\010\n\004root\0200\022\004node\030\002\"\000(\001@\000J\000" timestamp:<wall_time:163842..., err=<nil>
           234.309ms      0.030ms                    event:kv/kvserver/replica_gossip.go:121 [n=1,s=1,r=6/1:/Table/{SystemCon…-11}] gossiping system config
           143.134ms    -91.175ms                    structured:{"@type":"type.googleapis.com/cockroach.roachpb.ScanStats","numInterfaceSeeks":"2","numInternalSeeks":"2","numInterfaceSteps":"0","numInternalSteps":"2"}
           143.158ms      0.024ms                    structured:{"@type":"type.googleapis.com/cockroach.roachpb.ScanStats","numInterfaceSeeks":"2","numInternalSeeks":"2","numInterfaceSteps":"0","numInternalSteps":"2"}
           143.392ms      0.233ms                    structured:{"@type":"type.googleapis.com/cockroach.roachpb.ScanStats","numInterfaceSeeks":"2","numInternalSeeks":"2","numInterfaceSteps":"126","numInternalSteps":"88"}
           235.908ms    193.050ms    event:util/log/event_log.go:32 [n=1,client=127.0.0.1:37856,hostssl,user=root] "Timestamp":1638425372086132737,"EventType":"create_role","Statement":"CREATE ROLE rolea","Tag":"CREATE ROLE","User":"root","RoleName":"rolea"
           235.947ms      0.040ms    event:sql/catalog/descs/leased_descriptors.go:234 [n=1,client=127.0.0.1:37856,hostssl,user=root] releasing 0 descriptors
           235.973ms      0.026ms    event:sql/conn_executor_exec.go:1849 [n=1,client=127.0.0.1:37856,hostssl,user=root] AutoCommit. err: <nil>
        
        (above trace from test CreateRole/create_role_with_no_options. got 19, expected 18)
        --- FAIL: TestBenchmarkExpectation/CreateRole/create_role_with_no_options (17.07s)
=== RUN   TestBenchmarkExpectation/CreateRole
    --- FAIL: TestBenchmarkExpectation/CreateRole (17.94s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #73168 bench/rttanalysis: TestBenchmarkExpectation failed [C-test-failure O-robot branch-release-21.2]
</p>
</details>
/cc @cockroachdb/sql-schema
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBenchmarkExpectation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/bench/rttanalysis/testdata/benchmark_expectations
+++ b/pkg/bench/rttanalysis/testdata/benchmark_expectations
@@ -30,7 +30,7 @@ exp,benchmark
 17,CreateRole/create_role_with_1_option
 19,CreateRole/create_role_with_2_options
 20,CreateRole/create_role_with_3_options
-18,CreateRole/create_role_with_no_options
+18-19,CreateRole/create_role_with_no_options
 26,DropDatabase/drop_database_0_tables
 30-35,DropDatabase/drop_database_1_table
 36-42,DropDatabase/drop_database_2_tables

